hack/install-go.sh

```diff
@@ -2,7 +2,26 @@
 
 destination=$1
 version=$(grep "^go " go.mod |awk '{print $2}')
-tarball=go$version.linux-amd64.tar.gz
+
+arch="$(uname -m)"
+
+case $arch in
+    x86_64 | amd64)
+        arch="amd64"
+        ;;
+    aarch64 | arm64)
+        arch="arm64"
+        ;;
+    s390x)
+        arch="s390x"
+        ;;
+    *)
+        echo "ERROR: invalid arch=${arch}, only support x86_64, aarch64 and s390x"
+        exit 1
+        ;;
+esac
```

Comment:
lets see please if we do need it imo, because there wasnt an item to support ppc and i prefer to not do just parts unless needed

on the other hand, the suggestion is indeed more compact and robust, so it will be fine to adapt this
but the rest shouldn't be adapted yet

thanks

Reply:
I actually prefer the more verbose version, since it is more easy to read later on and to see what is intended to work.

My proposal would be i keep the switch-case, but change the default to a warning instead of an error.

Reply:
i do understand it is more verbose, but it is good to be in the same format on the repos we maintain 
anyhow i wont insist, we can change later according needs

we do need to error imo if one supplied unsupported version in case we keep it this way

Thanks